src/System/Extensions/XmlExtensions.cs

```diff
@@ -117,7 +117,7 @@ public static bool IsWellFormedXml(this Stream stream, CompressionType compressi
 				}
 			}
 
-			XmlReaderSettings settings = new XmlReaderSettings();
+			XmlReaderSettings settings = new XmlReaderSettings() { XmlResolver = null };
```

Comment:
FYI, in these constructor calls (throughout the file), you can exclude the `()`

Reply:
Fixed, thanks